ui/index.d.ts

```diff
@@ -1205,7 +1205,8 @@ export declare function addExtensionUserAgent(client: IAddUserAgent): void;
  */
 export function createAzureClient<T extends IAddUserAgent>(
```

Comment:
> For my use case, I wanted to be able to create a client that used a different acceptLanguage option for localization reasons.

I assume you just pass along the VS Code language? I'd actually prefer if you just added that functionality inside `createAzureClient`. The point of the method is to ensure best practices, and localization fits that bill

Reply:
Yes that's what I was going for